packages/react-reconciler/src/ReactFiberBeginWork.new.js

```diff
@@ -2338,7 +2339,9 @@ function updateDehydratedSuspenseComponent(
     // Register a callback to retry this boundary once the server has sent the result.
     registerSuspenseInstanceRetry(
       suspenseInstance,
-      retryDehydratedSuspenseBoundary.bind(null, current),
+      Schedule_tracing_wrap(
```

Comment:
Actually, let's only do this if the `enableSchedulerTracing` is on. Like so:

https://github.com/facebook/react/blob/master/packages/react-reconciler/src/ReactFiberCommitWork.old.js#L1822

We don't pull in a dependency on `scheduler/tracing` in the production build. Only in the profiling build.